### PR TITLE
Removal of useless line

### PR DESCRIPTION
**Pull request by Kartones**
@juanignaciosl CR please

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2101/))

File: app/controllers/api/json/imports_controller.rb [hunk 1]
```diff
@@ -318,7 +318,6 @@ def decorate_default_visualization_data!(data, data_import)
 
     if data_import.create_visualization && !data_import.visualization_id.nil?
       derived_vis = CartoDB::Visualization::Member.new(id: data_import.visualization_id).fetch
-      derived_vis_id = derived_vis.id unless derived_vis.nil?
```
Comment (juanignaciosl) [inline app/controllers/api/json/imports_controller.rb]: Not so fast! =D You deleted `derived_vis_id`, which is initialized to nil but used afterwards! Either you change line 324 as well (thus also deleting 317) or you only delete `unless ...`

**Comment by Kartones** [comment]
sorry, not feeling well today (but I hope tests will catch it)

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2103/))

**Pull request merged**